test/retorts.py

```diff
@@ -51,3 +65,114 @@ def __exit__(self, exc_type, exc_val, exc_tb):
         for mock_response in self.mock_responses:
             self.request_mock.remove(mock_response)
         self.request_mock.passthru_prefixes = self.passthru_prefixes
+
+
+class AuthResponseHelper(ResponsesHelper):
+    def __init__(self, passthru_url: str = None, request_mock: responses.RequestsMock = None) -> None:
+        super().__init__(request_mock)
+        self.passthru_url = passthru_url
+
+    def __enter__(self):
+        context = super().__enter__()
```

Comment:
You should return `context`.

Reply:
@hannes-ucsc `return context` is already on [line 103](https://github.com/DataBiosphere/azul/pull/685/files/5378ac6a7c9d1ca0b61f711b7d34e0483fdaf9e7#diff-225b710c3efb09eae6db97fdd83cf528R103). If it isn't what it should be, could you clarify what you meant by returning `context`?

Reply:
You are right. My mistake.